### PR TITLE
Add wikipedia search module (!wiki). gitignore logs/ directory

### PR DESCRIPTION
**Pull request by trustieee**
* Add wikipedia search module (!wiki)
* gitignore logs/ directory

### CODE REVIEW

File: Modix/Services/Wikipedia/WikipediaService.cs [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+        public async Task<WikipediaResponse> GetWikipediaResultsAsync(string phrase)
+        {
+            string query = string.Join(" ", phrase);
+            query = WebUtility.UrlEncode(query);
+            var client = new HttpClient();
+            var response = await client.GetAsync(string.Format(WikipediaApiScheme, query));
+
+            if (!response.IsSuccessStatusCode)
+            {
+                throw new WebException("Something failed while querying the Stack Exchange API.");
```
Comment (mastorm) [inline Modix/Services/Wikipedia/WikipediaService.cs]: No, I dont think so 💃  😆

File: Modix/Modules/WikipediaModule.cs [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+{
+    [Name("Wikipedia"), Summary("Search Wikipedia from Discord!")]
+    public class WikipediaModule : ModuleBase
+    {
+        [Command("wiki"), Summary("Returns a Wikipedia page result matching the search phrase.")]
+        public async Task Run([Remainder] string phrase)
+        {
+            var response = await new WikipediaService().GetWikipediaResultsAsync(phrase);
+
+            if (response != null && response.Query != null && response.Query.Pages.Any())
```
Comment (mastorm) [inline Modix/Modules/WikipediaModule.cs]: Can we invert this to remove nesting?

File: Modix/Modules/WikipediaModule.cs [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+                    else
+                    {
+                        messageBuilder.AppendLine(page.Extract);
+                    }
+                }
+
+                var message = messageBuilder.ToString();
+
+                if (message.Length > 0)
+                {
```
Comment (mastorm) [inline Modix/Modules/WikipediaModule.cs]: Can we invert this to remove nesting?

File: Modix/Modules/WikipediaModule.cs [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+                    if (message.Length > DiscordConfig.MaxMessageSize)
```
Comment (mastorm) [inline Modix/Modules/WikipediaModule.cs]: Can we invert this to remove nesting?

File: Modix/Modules/WikipediaModule.cs [hunk 5]
```diff
@@ -0,0 +1,69 @@
...
+                    {
+                        // How many batches do we need to send?
+                        // IE: 5000 / 2000 = 2.5
+                        // Round up = 3
+                        decimal batchCount = Math.Ceiling(decimal.Divide(message.Length, DiscordConfig.MaxMessageSize));
+                        int cursor = 0;
```
Comment (mastorm) [inline Modix/Modules/WikipediaModule.cs]: What exactly is a cursor in that case? Im really tired, so i might just be a little to stupid, but i just cant get behind what its doing. Maybe rename it?

**Review by mastorm — CHANGES_REQUESTED** [review]

File: Modix/Services/Wikipedia/WikipediaService.cs (hunk 1 above)
Comment (khionu) [reply Modix/Services/Wikipedia/WikipediaService.cs]: To note, this issue didn't get addressed.

**Pull request merged**